data/events/scripts/player.lua

```diff
@@ -74,145 +79,86 @@ function Player:onLookInBattleList(creature, distance)
 			description = string.format("%s\nIP: %s", description, Game.convertIpToString(creature:getIp()))
 		end
 	end
-	self:sendTextMessage(MESSAGE_INFO_DESCR, description)
+	local ret = hasEventCallback(EVENT_CALLBACK_ONLOOKINBATTLELIST) and EventCallback(EVENT_CALLBACK_ONLOOKINBATTLELIST, self, creature, distance, description) or description
+	self:sendTextMessage(MESSAGE_INFO_DESCR, ret)
 end
 
 function Player:onLookInTrade(partner, item, distance)
-	self:sendTextMessage(MESSAGE_INFO_DESCR, "You see " .. item:getDescription(distance))
+	local description = "You see " .. item:getDescription(distance)
+	local ret = hasEventCallback(EVENT_CALLBACK_ONLOOKINTRADE) and EventCallback(EVENT_CALLBACK_ONLOOKINTRADE, self, partner, item, distance, description) or description
+	self:sendTextMessage(MESSAGE_INFO_DESCR, EventCallback(EVENT_CALLBACK_ONLOOKINTRADE, self, partner, item, distance, description))
```

Comment:
looking at the previous example, seems like you should use ret instead you call the function again.
`self:sendTextMessage(MESSAGE_INFO_DESCR, ret)`